### PR TITLE
Add automated deployments

### PR DESCRIPTION
**Pull request by marvinbuss**
* Add automated deployments for better testing and validation
* Format all terraform files with `terraform fmt -recursive`
* Add missing identity for ADF
* Add missing role assignments for ADF
* Remove Git connection for ADF
* Update file structure to align with common standard structure for TF projects
* Add Function Keys to ADF deployments

### CODE REVIEW

**Comment by marvinbuss** [comment]
Workflow fails because the subscription is currently disabled.

**Review by esbran — DISMISSED** [review]

**Review by uday31in — APPROVED** [review]

File: infrastructure/config/UdayTenant/vars.tfvars [hunk 1]
```diff
@@ -0,0 +1,3 @@
+location    = "swedencentral"
```
[the remark this answers is not in the document]
Comment (marvinbuss) [reply infrastructure/config/UdayTenant/vars.tfvars:1]: This is used for e2e deployment and tests. Shall I remove tests?

[the remark this answers is not in the document]
Comment (marvinbuss) [reply infrastructure/config/UdayTenant/vars.tfvars:1]: This relies on the configuration: 
image

[the remark this answers is not in the document]
Comment (marvinbuss) [reply infrastructure/config/UdayTenant/vars.tfvars:1]: I can also switch back to the other tenant or use a different name. Let me know what you prefer.

**Review by krnese — APPROVED** [review]

**Pull request merged**